CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ foreach(p
   endif()
 endforeach()
 
-option(BENCHMARK_ENABLE_TESTING "Enable testing of the benchmark library." ON)
+option(BENCHMARK_ENABLE_TESTING "Enable testing of the benchmark library." OFF)
```

Comment:
I'm a little rusty but I think the "right" way to do this is at the subproject level:

https://stackoverflow.com/questions/14061605/override-option-in-cmake-subproject
https://stackoverflow.com/questions/3766740/overriding-a-default-option-value-in-cmake-from-a-parent-cmakelists-txt

Reply:
Hey Andrey! We currently need to disable this because it simply doesn't work. Once we figure out the best way to have the test running in an isolated environment, we'll make it toggleable. Thanks for the links.